libs/agno/agno/models/anthropic/claude.py

```diff
@@ -112,6 +112,7 @@ def _format_file_for_message(file: File) -> Optional[Dict[str, Any]]:
                 "type": "url",
                 "url": file.url,
             },
+            "citations": {"enabled": True},
```

Comment:
We enabled by default, because it is only used when there is an uploaded document and doesn't count towards output tokens (essentially free)